python/setup.py

```diff
@@ -289,6 +289,8 @@ def get_packages(self):
         "frozenlist",
         "requests",
         "virtualenv",  # For pip runtime env.
+        # We need to discuss if there is other alternative for converting dict to dataclass.
+        "dacite",
```

Comment:
I import `dacite` to achieve converting `dict` to `dataclass`. We can also find a better way to do this if possible.